docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/google-tag-manager-custom-template/v3-tags/tag-template-guide/index.md

```diff
@@ -1,10 +1,12 @@
 ---
-title: "Tag guide"
+title: "Snowplow v3"
 date: "2021-11-18"
-sidebar_position: 200
+sidebar_position: 100
 ---
 
-This template implements the Snowplow JavaScript tracker v3. The template supports all the features of the tracker, with a few exceptions due to the limitations of custom templates’ sandboxed JavaScript.
+This template implements the Snowplow JavaScript tracker v3.
```

Comment:
Should we link back up to the web tracker or is it clear enough since it lives under that tracker in the navigation?